client/src/plugins/usage-statistics/event-handlers/VersionInfoLinkOpenedEventHandler.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import BaseEventHandler from './BaseEventHandler';
+
+const VERSION_INFO_OVERLAY_ID = 'version-info-overlay';
+const LINK_SELECTOR = `#${VERSION_INFO_OVERLAY_ID} a[href]`;
+
+/**
+  * Sends event when user opens a link from the version info overlay.
+ */
+export default class VersionInfoLinkOpenedEventHandler extends BaseEventHandler {
+
+  constructor(props) {
+    const {
+      onSend
+    } = props;
+
+    super('versionInfoLinkOpened', onSend);
+  }
+
+  // In Chromium, a click event is dispatched also when user activates a link with the keyboard.
+  onAfterEnable() {
+    document.addEventListener('click', this.handleLinkOpened);
+  }
+
+  onAfterDisable() {
+    document.removeEventListener('click', this.handleLinkOpened);
+  }
+
+  /**
+   *
+   * @param {MouseEvent} event
+   */
+  handleLinkOpened = event => {
+    const { target } = event;
+
+    if (!target.matches || !target.matches(LINK_SELECTOR)) {
```

Comment:
Which targets do not contain a `matches`?

Reply:
Any `EventTarget` which is not an `Element`. However, it's unnecessary here as it's hard to imagine click events from non-DOM elements.

Reply:
That `EventTarget` would be, on a HTML page what? Would help me to better understand it.

To make the whole thing even safer we could intercept link clicks (via DOM delegation) on the VersionInfoOverlay only. Then these kinds of questions would not arise.

Reply:
> To make the whole thing even safer we could intercept link clicks (via DOM delegation) on the VersionInfoOverlay only

That's what we are doing with `target.matches(LINK_SELECTOR)`. 

> That EventTarget would be, on a HTML page what? Would help me to better understand it.

As I said - it's unnecessary and I will remove that :)

Reply:
The alternative I had in mind:

Add a `click` listener on the overlay. Clicks bubble up, cf. https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/click. 

On that listener `target` remains the link event. You do not have to register / de-register global click events in this case, as all intercepting happens locally in the component.

Reply:
Note that overlay appears in the DOM only when it's open. When you close the overlay, it's removed from the DOM. Thus, the component would have to know that the overlay has been opened to register for the click events. This means the VersionInfo component would have to take over listening for the click events inside it to propagate the information further to the event handler. I decided against this is solution as I wanted to encapsulate the link open event handler.

> Clicks bubble up, cf. https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/click.\

Correct, we use it in the current solution. The click event listener is [registered for the whole document](https://github.com/camunda/camunda-modeler/pull/2317/files#diff-287ef8bed73cfc09839844737e49c1318d9951f538caf3fb14124a90ccc4528cR30-R36). Then, when a click event bubbles up, we check whether the click comes from the version info overlay and specifically an anchor tag with href (this is what `matches` part is about).

Reply:
Ah, makes sense. 

So the telemetry handler is not being called from the version info overlay. The version info dialog does not know about that stuff existing _at all_.

In that case the solution makes more sense to me. Looking into the future I could think of making that "intercept a click and log it" a generic concern, i.e. we could tag our UI elements we want to intercept clicks on via data attributes. Just in case we see more of these things happening, of course.